packages/terra-table/docs/TableCell.md

```diff
@@ -0,0 +1,25 @@
+# Terra Table Cell
+
+The Terra Table Cell is a sub component that when used in conjuction with a table row can horizontally align child content.
```

Comment:
I would remove the can horizontall align child content. I think saying that it's a Cell used inside a row is fine enough, and it reduces our documentation maintenance if we don't need to rewrite descriptions.

Reply:
Updated.